test/service.ts

```diff
@@ -108,13 +128,14 @@ const fakeGrpcProtoLoader = {
 
 describe('GrpcService', () => {
   // tslint:disable-next-line:variable-name
-  let GrpcServiceCached;
-  // tslint:disable-next-line:variable-name
-  let GrpcService;
-  let grpcService;
+  let GrpcServiceCached: typeof serviceTypes.GrpcService;
+  // tslint:disable-next-line:variable-name no-any
+  let GrpcService: any;
```

Comment:
What happens when we set this to `GrpcService: GrpcService`?

Reply:
Because, from my experience, setting these variables to `any` later causes the Google team to highlight an error. Again, common-grpc might be different so I'd defer to your better judgement

Reply:
its upset the typescript and give me error and force me to cast it to any respectively, for private functions!

Reply:
Do you think maybe we should take that approach then? Annotate `GrpcService: GrpcService` then cast all of the test file functions to their `src` file interfaces?

Reply:
Yes agree on approach annotate GrpcService: GrpcService, but for private methods typescript is not happy and in several of my PR's Justin suggested me to avoid behavioral changes and use "any" type!